gui/.gitignore

```diff
@@ -0,0 +1,45 @@
+lib-cov
+*.seed
+*.log
+*.csv
+*.dat
+*.out
+*.pid
+*.gz
+*.swp
+
+pids
+logs
+results
+tmp
+
+# Build
+public/css/main.css
+
+# Coverage reports
+coverage
+
+# API keys and secrets
+.env
+
+# Dependency directory
+node_modules
+bower_components
+
+# Editors
+.idea
```

Comment:
These shouldn't exist under GUI. Always open Luna under the `luna` root directory, not the `gui` directory.